### PR TITLE
Wall fixes

### LINKED ISSUE #558: Wall creation UI date picker right and left arrows missing
As shown in this screenshotThe left and right arrow icons in the date picker UI are not showing up. I'm using Chrome 47 on Windows 10, User-Agent string `"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.111 Safari/537.36"`.

### PR DESCRIPTION
**Pull request by aneeshd16**
Fixes #558, #562 and other wall creation issues.

Requires a `npm install` to update angular-loading-bar

### CODE REVIEW

**Comment by aneeshd16** [comment]
@sudheesh001 can you please review this?

File: app/js/controllers/wallDisplay.js [hunk 1]
```diff
@@ -43,8 +43,8 @@ function WallDisplay($scope, $stateParams, $interval, $timeout, $location, $http
         	}
         }
 
-        if (vm.wallOptions.layoutStyle === '4') {
-            if (term === "") {
+        if (vm.wallOptions.layoutStyle == '4') {
+            if (term == "") {
```
Comment (sudheesh001) [inline app/js/controllers/wallDisplay.js]: Use `===` only, we don't want to break the `jshint` standard

Comment (aneeshd16) [inline app/js/controllers/wallDisplay.js]: @sudheesh001 agreed. However, using `===` seems to be breaking things. Till we figure exactly what is breaking, we can proceed with this.

**Comment by aneeshd16** [comment]
closing this, will be opening a new PR.

**Pull request closed without merging**